salsa_dancing_molecules/simulations/nve.py

```diff
@@ -35,16 +38,26 @@ def run_for_materials(formula, api_key, steps, output_path, repeat=0):
         if repeat > 0:
             atoms = atoms.repeat(repeat)
 
+        if potential == '':
```

Comment:
Is this really needed if it states in the command help that the default is Lennard Jones?

Reply:
Removed it!